i2c_comms/i2c_receiver.py

```diff
@@ -0,0 +1,25 @@
+# I2C COMMUNICATIONS TEST (RASPBERRY PI)
+# Gets potentiometer readings from Arduino (and later an ADC when we get it).
+# Code to read values will need to change to match the format the ADC will use
+# to send its data.
+
+from smbus2 import SMBus
+
+# I2C address hould be the same on the other device.
```

Comment:
Does pot count also need to be the same? I would assume. If so maybe mention in that comment